pkg/flink/internal/store/store_test.go

```diff
@@ -41,9 +40,9 @@ func tokenRefreshFunc() error {
 
 func (s *StoreTestSuite) TestGenerateStatementName() {
 	statementRegex := `^cli-\d{4}-\d{2}-\d{2}-\d{6}-[a-f0-9]{8}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{12}$`
-	rapid.Check(s.T(), func(t *rapid.T) {
+	for i := 0; i < 100; i++ {
```

Comment:
Why 100? Can we decrease this to make this test faster?

Reply:
sure, reduced it to 10